### DISCUSSION TITLE
Keine Einweisungsteilnehmer entfernen, die bereits unterschrieben haben

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#80] [Ideas]
"Wenn ich eine Einweisung digital durchführe und nur ein Teil der ausgewählten Mitarbeiter zur Einweisung erscheint und unterschreibt (was sehr häufig vorkommt), kann ich beim Entfernen der Mitarbeiter auch diese entfernen die bereits unterschrieben haben.
Ich hoffe da gibt es eine technische Lösung für, das diese erst nach entfernen der Unterschrift gelöscht werden können."

https://github.com/Samedis-care/samedis-care-community/assets/80519295/1f0807ea-5a40-4b1d-97bf-52d6cf3a8b60

**Comment by SCjona** [comment]
Optionen:
1. Lösch Knopf neben der Signatur
2. Beim Entfernen eines Mitarbeiters kann ein anderer Bestätigungsdialog kommen wenn eine Unterschrift vorhanden ist
3. Das Entfernen eines Mitarbeiters mit Unterschrift kann pauschal deaktiviert werden

Würde Optionen 1 und 2 implementieren, 3 kann zu Verwirrung führen ("Warum ist das deaktiviert?")

**Reply by moritzvolkmar** [reply]
Option 1 & 2 ist eine gute Idee.
